tests/integration/cgroups.bats

```diff
@@ -155,14 +165,22 @@ function setup() {
 
 	dd if=/dev/zero of=backing.img bs=4096 count=1
 	dev=$(losetup --find --show backing.img) || skip "unable to create a loop device"
+	LOOPBACK_DEVICES+=("$dev") # cleanup in teardown_loopback
```

Comment:
shellcheck doesn't like this. I guess we can switch to writing a list of filenames to a file instead...